### PR TITLE
Minor DX quality of life improvements

### PR DESCRIPTION
**Pull request by millnut**
Some minor changes that will hopefully improve other developers' experience when working locally.

1. Ignore macOS files in .gitignore
2. Change the Lando scanner to only check if the service is up 5 times, rather than the default 25
3. Fix deprecation in the settings file when running Lando
4. Add base themes to phpcs checks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,3 +77,7 @@
 .ddev/homeadditions/
 .ddev/providers/
 .ddev/xhprof/
+
+# Ignore macOS DS_Store files
+.DS_Store
+**/.DS_Store
```
Comment (andybroomfield) [inline .gitignore:83]: Whilst we certinally should ignore these, should we instead encourage devs to[ set up a global .gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)?

File: phpcs.xml.dist [hunk 2]
```diff
@@ -3,10 +3,12 @@
   <description>PHP CodeSniffer configuration for LocalGovDrupal.</description>
 
   <arg name="extensions" value="inc,install,module,php,profile,test,theme,yml"/>
-  <config name="drupal_core_version" value="9"/>
+  <config name="drupal_core_version" value="10"/>
 
   <file>web/modules/contrib/</file>
   <file>web/profiles/contrib/localgov/</file>
+  <file>web/themes/contrib/localgov_base/</file>
+  <file>web/themes/contrib/localgov_scarfolk/</file>
```
Comment (stephen-cox) [inline phpcs.xml.dist]: Good catch to include these. Should be use something like we do for modules; include the `web/themes/contrib/` and then exclude anything that doesn't start with `localgov_`? This way we catch any LGD themes.

Comment (millnut) [reply phpcs.xml.dist]: Yep that's a great suggestion @stephen-cox I'll update the PR and let you know when it's ready for review again

Comment (millnut) [reply phpcs.xml.dist]: Hi @stephen-cox I've updated the matching

File: .gitignore (hunk 1 above)
Comment (millnut) [reply .gitignore:83]: Yes, I agree @andybroomfield that we should encourage global `.gitignore` usage; maybe we can add this somewhere in the getting started development documentation.

This change will also act as a failsafe should users not do the step.

**Review by stephen-cox — APPROVED** [review]
Looks good to me :+1:

File: phpcs.xml.dist (hunk 2 above)
Comment (stephen-cox) [reply phpcs.xml.dist]: Thanks @millnut - I've just approved this

**Pull request merged**